README.md

```diff
@@ -40,19 +40,27 @@ Available gadgets:
 		Formatters:
 			BinaryFormatter, LosFormatter, ObjectStateFormatter, SoapFormatter
 
+	DataSet (DataSet gadget) [derived method]
+		Formatters:
+			BinaryFormatter, LosFormatter, ObjectStateFormatter, SoapFormatter
+
 	ObjectDataProvider (ObjectDataProvider gadget)
 		Formatters:
-			DataContractSerializer, FastJson, FsPickler, JavaScriptSerializer, Json.Net, Xaml, XmlSerializer, YamlDotNet < 5.0.0
+			DataContractSerializer, DataContractSerializer2, FastJson, FsPickler, JavaScriptSerializer, Json.Net, Xaml, Xaml2, XmlSerializer, YamlDotNet < 5.0.0
```

Comment:
What is `DataContractSerializer2` and `Xaml2`?

Reply:
Xaml2 uses ResourceDictionary in addition to just using ObjectDataProvider as in Xaml

datacontractserializer2 uses Xaml.Parse rather than using ObjectDataProvider directly which is useful for bypassing blacklists

These were the shortest name I could come up with but I am open to suggestions on this. I didn't want to add a new argument called "variant" as I didn't want to make the generators longer by having another input parameter. The same thing can occur when we need a payload for BinaryFormatter - I have changed everything to use the shortest payload we have at the moment but it might be blocked and in that case, we should change the code to generated an expected payload (I am totally fine with that although not being user friendly).

Reply:
Ok, thanks, that fine. Perhaps just add a note like that one to these serializers if its not already there

Reply:
Done